scripts/analyze.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+# Copyright 2014-2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import matplotlib.pyplot as pyplot
+import numpy
+
+import argparse
+
+parser = argparse.ArgumentParser(description="Plot rttest output")
```

Comment:
We should use single quotes consistently wherever possible.

Same in below and in the next file.